.github/workflows/build-test.yml

```diff
@@ -196,6 +194,12 @@ jobs:
     environment: Default
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - run: git checkout dev
```

Comment:
Please add a brief comment why these checkouts & fetch depth is necessary, in case the commit information is lost